reference/catalog.v3.yml

```diff
@@ -903,8 +1158,8 @@ paths:
         - 250 characters product name length.
 
         **Usage Notes**
-        * `POST` requests to `/products` accept a `video` array. 
-        * `POST` requests to `/products/{product_id}/videos` accept a `video` object.
+        * `POST` requests to `/products` accept a single `video` object. To send an array of video objects, use: `/products/{product_id}/videos`.
+        * You can supply category IDs, but the channel ID array should remain empty.
```

Comment:
elsewhere we've changed this. do we want to keep the `channel ID array should remain empty` in this location?